frontend/app/src/store/history/index.ts

```diff
@@ -667,6 +668,82 @@ export const useTransactions = defineStore('history/transactions', () => {
     }
   };
 
+  const addTransactionEvent = async (
+    event: NewEthTransactionEvent
+  ): Promise<ActionStatus> => {
+    let success = false;
+    let message = '';
+    try {
+      await api.history.addTransactionEvent(event);
+      success = true;
+    } catch (e: any) {
+      message = e.message;
+    }
+
+    await Promise.all([fetchTransactions()]);
```

Comment:
Since it's a single one is there a difference between this and doing `await fetchTransactions()`